poros/game_results.rb

```diff
@@ -0,0 +1,64 @@
+
+class GameResults
+  
+  def service 
+    GameService.new
+  end
+
+  def game_params_by_name(game_name)
+    params = service.get_games_by_name(game_name)[0]
+    binding.pry
+    result = {data: {
+      age_ratings:  run_array(:get_age_ratings, :rating, params[:age_ratings]), # array of ids
+      release_date: service.get_release_date(params[:first_release_date]), # id
+      cover:        cover_url(params[:cover]),
+      popularity:   params[:popularity], # good
+      summary:      params[:summary], # good
+      name:         params[:name], # good
+      total_rating: params[:total_rating], # good
+      categories:   params[:category], # got 0
+      genres:       run_array(:get_game_genres, :name, params[:genres]), # array of ids
+      keywords:     params[:keywords], # array of ids -- several hundred :(
+      platforms:    run_array(:get_platforms, :name, params[:platforms]), # array of ids
```

Comment:
other return values are full arrays of ids `[1,10,43...]`. We can then hit the API again to return the details of a single id. If there are many ids, then I use the method `run_array` (general, I know) to iterate through that array of ids and return the name of each